### PR TITLE
draft of the landing page

### PR DESCRIPTION
**Pull request by egurung11**
this is the outline for the landing page, full info not added yet

### CODE REVIEW

**Comment by wkg7vb** [comment]
looks good

File: BAR/BAR.csproj [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="8.0.8" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.8" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="8.0.8" />
+    <PackageReference Include="Install-PackageBlazor.Bootstrap" Version="3.0.0" />
```
Comment (max-dejesus) [inline BAR/BAR.csproj:15]: may cause error

File: BAR/Components/Routes.razor [hunk 2]
```diff
@@ -9,3 +9,4 @@
         <FocusOnNavigate RouteData="routeData" Selector="h1" />
     </Found>
 </Router>
+s
```
Comment (max-dejesus) [inline BAR/Components/Routes.razor:12]: ?

**Review by haley-v — APPROVED** [review]
cool beans!

File: BAR/Components/Pages/Landing.razor [hunk 3]
```diff
@@ -0,0 +1,66 @@
+﻿@page "/landing"
```
Comment (max-dejesus) [inline BAR/Components/Pages/Landing.razor:1]: small thing but this will be root eventually.

**Review by wkg7vb — APPROVED** [review]
looks good

**Pull request merged**